package/dnf.spec.in

```diff
@@ -12,7 +12,7 @@
 
 Name:		dnf
 Version:	@DNF_MAJOR@.@DNF_MINOR@.@DNF_PATCH@
-Release:	2%{?dist}
+Release:	2%{?dist}%{?extra_rel}
```

Comment:
I don't see any difference in you comment. You meant ```2.fc21.abcdef``` in the second case?
I prefer the one that is aligned with packaging guidelines
https://fedoraproject.org/w/index.php?title=Packaging:NamingGuidelines&oldid=400817#Snapshot_packages
Otherwise it cannot be used for nightly builds since there was already an fight on fedora-devel that our snapshots did not match the guidelines.